src/v2/view-builder/views/device/DeviceChallengePollView.js

```diff
@@ -64,17 +57,35 @@ const Body = BaseFormWithPolling.extend(
         this.doLoopback(deviceChallenge.domain, deviceChallenge.ports, deviceChallenge.challengeRequest);
         break;
       case Enums.CUSTOM_URI_CHALLENGE:
-        this.title = loc('oktaVerify.button', 'login');
+        this.title = loc('customUri.title', 'login');
         this.add(View.extend({
           className: 'skinny-content',
           template: hbs`
             <p>
-              {{{i18n code="customUri.required.content.p1" bundle="login"}}}
-            </p>
-            <p>
-              {{{i18n code="customUri.required.content.p2" bundle="login" arguments="downloadOVLink"}}}
+              {{i18n code="customUri.required.content.prompt" bundle="login"}}
             </p>
           `,
+        }));
+        this.add(createButton({
```

Comment:
if this is exact same code as in file `ChallengeOktaVerifyFastPassView.js` we should create a reusable button instead of duplicating the code.